indico/web/client/js/react/components/principals/ACLField.jsx

```diff
@@ -100,6 +101,20 @@ const ACLField = props => {
       text: r.name,
     }));
 
+  const onDropdownOpen = event => {
+    // Fixes semantic dropdown cropping within constrained elements
+    // `defer` ensures the scroll is queued after all state updates
+    const target = event.currentTarget;
+    if (scrollOnOpen) {
+      _.defer(() =>
+        target?.querySelector('.menu.visible')?.scrollIntoView({
```

Comment:
I'd use `if (scrollOnOpen && target)` above and remove the `?` after `target` here